### PR TITLE
Resolve with dirname

### LINKED ISSUE #828: Why isn't "eslint-import-resolver-node" listed in `eslint-module-utils`'s peerDependencies?
This plugin does not work with `pnpm` because it relies on NPM3's flat `node_modules` structure. Please see: https://github.com/pnpm/pnpm/issues/739, where there is a reproducible repo.

### PR DESCRIPTION
**Pull request by ishitatsuyuki**
Close #828

### CODE REVIEW

File: src/ExportMap.js [hunk 1]
```diff
@@ -262,7 +262,7 @@ function captureTomDoc(comments) {
 }
 
 ExportMap.get = function (source, context) {
-  const path = resolve(source, context)
+  const path = resolve(source, context, __dirname)
```
Comment (ljharb) [inline src/ExportMap.js]: this would be the dirname of this file - is that intended? I'd expect `process.cwd()`

**Comment by ishitatsuyuki** [comment]
@ljharb All typos fixed. Tests are passing now.

File: src/ExportMap.js (hunk 1 above)
Comment (ishitatsuyuki) [reply src/ExportMap.js]: This is where we load the "resolver module" from, not the actual module that we're resolving. I've updated the doc comments.

**Comment by zkochan** [comment]
Is there something else that needs to be done in order to get this merged?

**Comment by ljharb** [comment]
@zkochan other maintainers would need to review it.

**Comment by zkochan** [comment]
can we ping them? It would be really great to have this issue solved

@benmosher could you review this PR, please?

**Comment by donifer** [comment]
Friendly bump 🙏

**Comment by benmosher** [comment]
I don't think I like this solution but mostly I think I don't understand why capturing `__dirname` at the calling location and passing it to `resolve` is an ideal solution to #828. I also don't really understand how it even works at all.

**Comment by ljharb** [comment]
@ishitatsuyuki if you're still interested in landing this PR, could you rebase it, address the comments, and mark this as ready for review?

**Comment by ishitatsuyuki** [comment]
Closing as I'm no longer interested in this.

**Pull request closed without merging**